packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -437,4 +437,5 @@ export default connect(({ client, notifications, ui, traders_hub }) => ({
     upgrade_info: client.upgrade_info,
     setSubSectionIndex: ui.setSubSectionIndex,
     sub_section_index: ui.sub_section_index,
+    is_high_risk_client_for_mt5: modules.cfd.is_high_risk_client_for_mt5,
```

Comment:
let's move `is_high_risk_client_for_mt5` and `can_get_more_cfd_mt5_accounts` to hooks.
anyways there are not being used in CFD package and only in Core and Appstore.